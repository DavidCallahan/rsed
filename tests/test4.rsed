print "ENV1=" $ENV1
print "ENV2=" $ENV2
print "ENVMISSING=" $ENVMISSING
