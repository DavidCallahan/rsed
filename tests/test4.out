ENV1=1
ENV2=ENV2 value
ENVMISSING=
